File: simple_interest.py
#!/usr/bin/env python3

# Simple Interest Calculator
# Formula: Simple Interest = (Principal * Rate * Time) / 100

def calculate_simple_interest(principal, rate, time):
    return (principal * rate * time) / 100

def main():
    print("===================================")
    print("  Simple Interest Calculator")
    print("===================================")
    
    try:
        principal = float(input("Enter Principal amount: "))
        rate = float(input("Enter Rate of Interest (%): "))
        time = float(input("Enter Time period (in years): "))
        
        simple_interest = calculate_simple_interest(principal, rate, time)
        
        print("===================================")
        print(f"Simple Interest = {simple_interest}")
        print("===================================")
    
    except ValueError:
        print("Error: Please enter valid numbers.")

if __name__ == "__main__":
    main()

File: simple-interest.js
// Simple Interest Calculator
// Formula: Simple Interest = (Principal * Rate * Time) / 100

const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

console.log("===================================");
console.log("  Simple Interest Calculator");
console.log("===================================");

rl.question("Enter Principal amount: ", (principal) => {
    rl.question("Enter Rate of Interest (%): ", (rate) => {
        rl.question("Enter Time period (in years): ", (time) => {
            const p = parseFloat(principal);
            const r = parseFloat(rate);
            const t = parseFloat(time);
            
            const simpleInterest = (p * r * t) / 100;
            
            console.log("===================================");
            console.log(`Simple Interest = ${simpleInterest}`);
            console.log("===================================");
            
            rl.close();
        });
    });
});
